src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -815,7 +815,7 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		tabContainer.appendChild(tabBorderTopContainer);
 
 		// Tab Editor Label
-		const editorLabel = this.tabResourceLabels.create(tabContainer, { hoverDelegate: this.getHoverDelegate() });
+		const editorLabel = this.tabResourceLabels.create(tabContainer, { hoverTargetOverrride: tabContainer });
```

Comment:
Don't we still need to pass delegate here?

Reply:
Actually, it is not needed because the label already provides the default hover behaviour and tabs don't need any special hover behaviour